.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*.{adoc, bat, groovy, html, java, js, jsp, kt, kts, md, properties, py, rb, sh, sql, svg, txt, vm, xml, xsd}]
+charset = utf-8
+
+[*.{groovy, java, kt, kts, vm, xml, xsd}]
```

Comment:
Same as above. `[*.{java, xml, sql}]` should be enough.

Reply:
I agree with you! I just referred to Spring's `.editorconfig` file. It can be adjusted here. Thank you!

Reply:
Hi @ijliym I cannot edit the files. Maybe because you deleted the branch?
I'm not sure if it works, but can you restore the fork/branch?

Reply:
I'm very sorry, I don't know after delete this, there will be an impact. I've added it again now, but I don't think it might work. If it doesn't work, please trouble you add it yourself. I just want to have a `.editorconfig` file in the project, not necessarily if I add it. 
Sorry again!

Reply:
GitHub does not recognize the new branch, but no worries.
Although it's a little bit irregular, I'll make the changes after merge. :)
Thanks for the contribution!

Reply:
You're welcome.